packit_service/worker/testing_farm.py

```diff
@@ -152,7 +152,7 @@ def _payload_install_test(self, build_id: int, chroot: str) -> dict:
             "test": {
                 "fmf": {
                     "url": TESTING_FARM_INSTALLABILITY_TEST_URL,
-                    "name": "/packit/install-and-verify",
+                    "name": "/packit/installation",
```

Comment:
just to make sure, it can stay in the `test` key?

Reply:
yep, that is right:

https://gitlab.com/testing-farm/api/-/merge_requests/19